### ISSUE TITLE
MongoDB

### ISSUE THREAD

**Issue by EliasVRG** [EliasVRG/myFinances#2]
# Aprendizagem

- [ ]  Criar tabelas e colunas

- [ ]  Como gravar usando linguagem Python

- [ ]  Como conectar com a aplicação MyFinances

**Comment by AdamMatar2191** [comment]
Ei, estou trabalhando no aplicativo, mas eu crio contas separadas do mongodb atlas, já que os dados não são meus, então eu poderia fornecer os detalhes da conta e você pode alterar a senha
